### PR TITLE
VAGOV-1471: Query and template for Administration on hub landing page

### PR DESCRIPTION
**Pull request by acabouet**
## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/components/hub-rail.drupal.liquid [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                    <h4>Follow Us</h4>
+                    <ul class="va-nav-linkslist-list social">
+                        {% assign socialLinks =  administration.fieldSocialMediaLinks.platformValues | jsonToObj %}
+                        {% assign socialPlatforms = socialLinks | keys %}
+                        {% for socialPlatform in socialPlatforms %}
+                            {% assign socialLink = socialLinks | get: socialPlatform  %}
+                            {%  if socialLink.value %}
+                                {% if socialPlatform = "youtube_channel" %}
+                                    <li>
+                                        <a href="http://youtube.com/channel/{{ socialLink.value }}" target="_blank" rel="noopener noreferrer">
```
Comment (jbalboni) [inline src/site/components/hub-rail.drupal.liquid]: Same comment as above about target and rel

**Review by jbalboni — APPROVED** [review]
Some minor comments, otherwise looks good

**Comment by jbalboni** [comment]
This is the build error:

screen shot 2019-02-06 at 1 02 47 pm

Might be related to me moving filters into a separate file?

**Comment by acabouet** [comment]
Moved the filter which fixed the build for me locally but still failing here. Gonna set up socks proxy to troubleshoot.

**Pull request merged**